lib/safe-pg-migrations/plugins/blocking_activity_logger.rb

```diff
@@ -78,6 +92,26 @@ def log_blocking_queries
       raise
     end
 
+    def output_blocking_queries(queries)
+      if SafePgMigrations.config.blocking_activity_logger_verbose
+        SafePgMigrations.say(
+          "Statement was being blocked by the following #{'query'.pluralize(queries.size)}:", true
+        )
+        SafePgMigrations.say '', true
+        queries.each do |query, start_time|
+          SafePgMigrations.say "#{format_start_time start_time}:  #{query}", true
+        end
+      else
+        queries.each do |start_time, locktype, mode, pid, transactionid|
+          SafePgMigrations.say format_start_time(start_time), true
+          SafePgMigrations.say "lock type: #{locktype || 'null'}", true
+          SafePgMigrations.say "lock mode: #{mode || 'null'}", true
+          SafePgMigrations.say "lock pid: #{pid || 'null'}", true
+          SafePgMigrations.say "lock transactionid: #{transactionid || 'null'}", true
```

Comment:
Do you think we could have all of these information on one line? We might have a lot of queries, I fear it becomes unreadable if we multiply the number of lines by 5.

Reply:
sure thing